ltrace/loong.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index a568f2f..4bc7b4d 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -14,6 +14,15 @@ makedepends=('dejagnu')
+ backup=('etc/ltrace.conf')
+ source=(https://sources.archlinux.org/other/$pkgname/$pkgname-$pkgver.tar.bz2)
+ sha256sums=('0e6f8c077471b544c06def7192d983861ad2f8688dd5504beae62f0c5f5b9503')
++prepare(){
++  cd "$srcdir/$pkgname-$pkgver"
++  wget -O ./config/autoconf/config.sub "git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD"
```

Comment:
Would you please consider to add it to source?

Reply:
Thanks for your sentence noticing me that my git isn't source, and I take all patches to the patches set.